.circleci/config.yml

```diff
@@ -134,6 +146,62 @@ jobs:
           (cd /tmp/build \
             && chown -R runner . \
             && su runner -c "ctest -V --label-regex quick --parallel 2")
+  "Ubuntu-18.04-cuda10.1-build-only":
+    docker:
+      - image: nvidia/cuda:10.1-devel-ubuntu18.04
+    steps:
+      - checkout
+      - run: |
+          apt-get -q update -y
+          apt-get -q install -y apt-transport-https ca-certificates git gnupg software-properties-common wget
+          wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | apt-key add -
+          apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
+          apt-get -q update -y
+      - run: git submodule sync
+      - run: git submodule update --init
+      - run: |
+          apt-get -q install -y \
+            cmake \
+            g++ \
+            gcc \
+            libboost-all-dev \
+            libeigen3-dev \
+            libopenmpi-dev \
+            llvm-7-dev \
+            openmpi-bin \
+            ssh
+
+          cmake -S . -B /tmp/build -DENABLE_DIST_GALOIS=ON -DENABLE_HETERO_GALOIS=ON
+          cmake --build /tmp/build --target input
+          cmake --build /tmp/build --parallel 2
+  "Ubuntu-18.04-cuda10.2-build-only":
+    docker:
+      - image: nvidia/cuda:10.2-devel-ubuntu18.04
+    steps:
+      - checkout
+      - run: |
+          apt-get -q update -y
+          apt-get -q install -y apt-transport-https ca-certificates git gnupg software-properties-common wget
+          wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | apt-key add -
+          apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
+          apt-get -q update -y
+      - run: git submodule sync
+      - run: git submodule update --init
+      - run: |
+          apt-get -q install -y \
+            cmake \
+            g++ \
+            gcc \
+            libboost-all-dev \
+            libeigen3-dev \
+            libopenmpi-dev \
+            llvm-7-dev \
+            openmpi-bin \
+            ssh
+
+          cmake -S . -B /tmp/build -DENABLE_DIST_GALOIS=ON -DENABLE_HETERO_GALOIS=ON
```

Comment:
-> cmake -S . -B /tmp/build -DGALOIS_ENABLE_DIST=ON -DGALOIS_ENABLE_GPU=ON

Reply:
Thanks for reminding me! Though this change only needed if this PR goes in after #199 . Otherwise it is actually correct as it stands.